app/build.gradle.kts

```diff
@@ -85,12 +99,15 @@ android {
     }
 
     kotlinOptions {
-        jvmTarget = "1.8"
+        jvmTarget = JavaVersion.VERSION_11.toString()
+    }
+    buildFeatures {
+        buildConfig = true
```

Comment:
What is the reason for this configuration? Does it have to do with AGP 8.0.0 not generating the BuildConfig class by default?